src/main/java/Duke.java

```diff
@@ -1,10 +1,260 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    /**
+     * Maximum number of Tasks that can be stored
+     */
+    public static final int MAX_TASKS = 100;
+
+    /**
+     * The input scanner for the program
+     */
+    public static final Scanner SCANNER_INPUT = new Scanner(System.in);
+
+    /**
+     * An array that stores all the tasks
+     */
+    public static Task[] taskList;
+
+    /**
+     * Stores the most recent input from the user in String format
+     */
+    public static String strInput = "";
+
+    /**
+     * "false" by default. "true" when session with duke is to be ended.
+     */
+    public static boolean hasSessionEnded = false;
```

Comment:
naming it isSessionToEnd might be better, because normally most booleans start with 'is'. But 'has' works just as well here, so maybe it's personal preference